Package.swift

```diff
@@ -96,29 +96,20 @@ let package = Package(
             name: "MapboxNavigationTests",
             dependencies: [
                 "MapboxNavigation",
-                "FBSnapshotTestCase",
                 "Quick",
                 "TestHelper",
                 "OHHTTPStubs",
                 "CarPlayTestHelper",
+                "SnapshotTesting",
             ],
             exclude: [
-                // Exclude snapshot testing tests because it is not clear how to configure FBSnapshotTestCase in SPM.
                 "Info.plist",
                 "MapboxNavigationTests-Bridging.h",
-                "BottomBannerSnapshotTests.swift",
-                "CarPlayCompassViewSnapshotTests.swift",
-                "GuidanceCardsSnapshotTests.swift",
-                "SnapshotTest+Mapbox.swift",
-                "NavigationViewControllerTests.swift",
+                "NavigationViewControllerTests.swift", // Crash on `UNUserNotificationCenter.current()`. This API doesn't work in SPM.
```

Comment:
You can add conditional compilation around tests that don't run using ```#if SWIFT_PACKAGE```

Reply:
Yep, although, Xcode based testing isn't functioning as far as I know. I'm bringing more tests back in subsequent PR and will revisit some tests based on this suggestion.